features/web-audio.yml

```diff
@@ -0,0 +1,216 @@
+  - api.AudioContext.resume
+  - api.AudioContext.suspend
+  - api.AudioDestinationNode
+  - api.AudioDestinationNode.maxChannelCount
+  - api.AudioListener
+  - api.AudioListener.forwardX
+  - api.AudioListener.forwardY
+  - api.AudioListener.forwardZ
+  - api.AudioListener.positionX
+  - api.AudioListener.positionY
+  - api.AudioListener.positionZ
+  - api.AudioListener.upX
+  - api.AudioListener.upY
+  - api.AudioListener.upZ
+  - api.AudioNode
+  - api.AudioNode.channelCount
+  - api.AudioNode.channelCountMode
+  - api.AudioNode.channelInterpretation
+  - api.AudioNode.connect
+  - api.AudioNode.context
+  - api.AudioNode.disconnect
+  - api.AudioNode.numberOfInputs
+  - api.AudioNode.numberOfOutputs
+  - api.AudioParam
+  - api.AudioParam.automationRate
+  - api.AudioParam.cancelAndHoldAtTime
+  - api.AudioParam.cancelScheduledValues
+  - api.AudioParam.defaultValue
+  - api.AudioParam.exponentialRampToValueAtTime
+  - api.AudioParam.linearRampToValueAtTime
+  - api.AudioParam.maxValue
+  - api.AudioParam.minValue
+  - api.AudioParam.setTargetAtTime
+  - api.AudioParam.setValueAtTime
+  - api.AudioParam.setValueCurveAtTime
+  - api.AudioParam.value
+  - api.AudioParamMap
+  - api.AudioScheduledSourceNode
+  - api.AudioScheduledSourceNode.ended_event
+  - api.AudioScheduledSourceNode.start
+  - api.AudioScheduledSourceNode.stop
+  - api.AudioWorklet
+  - api.AudioWorkletGlobalScope
+  - api.AudioWorkletGlobalScope.currentFrame
+  - api.AudioWorkletGlobalScope.currentTime
+  - api.AudioWorkletGlobalScope.registerProcessor
+  - api.AudioWorkletGlobalScope.sampleRate
+  - api.AudioWorkletNode
+  - api.AudioWorkletNode.AudioWorkletNode
+  - api.AudioWorkletNode.parameters
+  - api.AudioWorkletNode.port
+  - api.AudioWorkletNode.processorerror_event
+  - api.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.port
+  - api.BaseAudioContext
+  - api.BaseAudioContext.audioWorklet
+  - api.BaseAudioContext.createAnalyser
+  - api.BaseAudioContext.createBiquadFilter
+  - api.BaseAudioContext.createBuffer
+  - api.BaseAudioContext.createBufferSource
+  - api.BaseAudioContext.createChannelMerger
+  - api.BaseAudioContext.createChannelSplitter
+  - api.BaseAudioContext.createConstantSource
+  - api.BaseAudioContext.createConvolver
+  - api.BaseAudioContext.createDelay
+  - api.BaseAudioContext.createDynamicsCompressor
+  - api.BaseAudioContext.createGain
+  - api.BaseAudioContext.createIIRFilter
+  - api.BaseAudioContext.createOscillator
+  - api.BaseAudioContext.createPanner
+  - api.BaseAudioContext.createPeriodicWave
+  - api.BaseAudioContext.createStereoPanner
+  - api.BaseAudioContext.createWaveShaper
+  - api.BaseAudioContext.currentTime
+  - api.BaseAudioContext.decodeAudioData
+  - api.BaseAudioContext.destination
+  - api.BaseAudioContext.listener
+  - api.BaseAudioContext.sampleRate
+  - api.BaseAudioContext.state
+  - api.BaseAudioContext.statechange_event
```

Comment:
Same for the BaseAudioContext. My understanding is that it was introduced for the benefit of the OfflineAudioContext interface, which is a later addition too, and doesn't prevent using the API.

Reply:
Most of what's on `BaseAudioContext` is part of the origin API accessed through the `AudioContext` constructor. `AudioContext` was made to inherit from the new `BaseAudioContext` and a lot of things were moved, but it was a backwards compatible change. This note in BCD is about this:
https://github.com/mdn/browser-compat-data/blob/8e86a205cfde203fc91f5548c9ffe142ede2270d/api/BaseAudioContext.json#L16

It also looks like the status of the `BaseAudioContext` members here ends up being clamped to `BaseAudioContext`, despite that being worked around in BCD with partial implementation. I'll file an issue.

Reply:
Makes sense. Let's keep this in.

Reply:
I've filed https://github.com/web-platform-dx/web-features/issues/1314 now.